src/Solver/FFTPoissonSolver.h

```diff
@@ -70,15 +70,15 @@ namespace ippl {
     }  // namespace detail
 
     template <typename FieldLHS, typename FieldRHS>
-    class FFTPoissonSolver : public Electrostatics<FieldLHS, FieldRHS> {
+    class FFTPoissonSolver : public Poisson<FieldLHS, FieldRHS> {
```

Comment:
May not be needed in the current PR but would it make sense to rename this as "FFTOpenPoissonSolver" as this works only with open BCs and the naming is then also consistent with FFTPeriodicPoissonSolver. I agree that as Andreas said it is not very nice to have the BCs name also in the solver class but in my opinion these are very specific solvers and they are also fundamentally different as we already discussed (Integral Vs Differential form).

Reply:
I think conceptually it makes sense, and I can do it in this PR too, if everyone agrees.